event/src/bytes.rs

```diff
@@ -0,0 +1,69 @@
+//! Thin wrapper on a Vec<u8> that correctly serializes and deserializes as bytes as opposed to a
+//! sequence of u8 integers.
+use serde::{
+    de::{self, Visitor},
+    Deserialize, Serialize,
+};
+
+/// Sequence of byte values.
+#[derive(PartialEq, Default, Debug)]
+pub struct Bytes(Vec<u8>);
```

Comment:
isn't there already a `Bytes` type from the `bytes` crate?  Do we really need to make our own?

Reply:
Yes, this is about serde of the Bytes to be raw bytes not a sequence of u8s. The IPLD codecs are naive and assume a Vec<T> maps to an IPLD List of T. We need to special case where T is u8 so that a Vec<u8> maps to IPLD bytes type.